src/model/openthaigpt_pretraining_model/llama_thai_tokenizer/merge_tokenizer.py

```diff
@@ -0,0 +1,44 @@
+from transformers import LlamaTokenizer
+from sentencepiece import sentencepiece_model_pb2 as sp_pb2_model
+import sentencepiece as spm
+import os
+
+llama_tokenizer_dir = "tokenizer.model"
```

Comment:
Upper case for constants example `LLAMA_TOKENIZER_DIR`.

Reply:
done